automation/Dockerfile.fedora

```diff
@@ -15,7 +15,9 @@ RUN dnf -y install --setopt=install_weak_deps=False \
                    rpm-build git python3-devel python3-dbus python3-coveralls \
                    python3-pyyaml python3-jsonschema python3-setuptools \
                    NetworkManager-ovs openvswitch libibverbs python36 \
-                   python3-gobject-base dnsmasq radvd python3-tox python2 && \
```

Comment:
This changes needs to be a separate PR to update the container image.

Reply:
Done.